scriptworker/log.py

```diff
@@ -88,7 +88,8 @@ def get_log_filenames(context):
     Returns:
         tuple: log file path, error log file path
     """
-    log_file = os.path.join(context.config['task_log_dir'], 'task_output.log')
+    # XXX Even though our logs aren't live, Treeherder looks for live_backing.log to show errors in failures summary
+    log_file = os.path.join(context.config['task_log_dir'], 'live_backing.log')
```

Comment:
Thanks!
I think signing logging output goes to stderr, and stack traces as well.  A single unified log seems like a win.